AEPCore/Sources/eventhub/EventHub.swift

```diff
@@ -394,8 +414,8 @@ final class EventHub {
         self.eventQueue.add(event)
         // Hot path, avoid unnecessary string converstion of event
         if Log.logFilter >= .trace {
```

Comment:
Should we consider adding `logFilter` (or `logLevel`) to the `Logger` protocol so we don't need to access the public Log class?

Reply:
That makes sense. Adding a `logFilter` getter will make it easier to control the logs for each SDKInstanceLogger in the future. I’ll include this update in my next PR